maps/site53/items/cards_ids.dm

```diff
@@ -420,6 +420,22 @@
 	item_state = "Admin_ID"
 	job_access_type = /datum/job/captain
 
+// ERT CARDS
+
+/obj/item/card/id/mtf
+	name = " mobile task force ID"
+	desc = "A black ID. Looks like the person wearing this won't give it up easy."
+	icon_state = "adminlvl5"
+	item_state = "Admin_ID"
+	job_access_type = /datum/job/mtf
+
+/obj/item/card/id/physics
+	name = " military ID"
+	desc = "A dark purple ID. Looks like the person wearing this won't give it up easy."
+	icon_state = "securitylvl5"
```

Comment:
will security level 5 let em in the front door? I remember people talking about this and UNGOC probably shouldn't have door access at all, let alone to the front